modin/utils.py

```diff
@@ -87,23 +113,27 @@ def _replace_doc(
     if (
         source_doc.strip()
         and apilink
-        and "Pandas API documentation <" not in target_doc
+        and "`Pandas API documentation for " not in target_doc
```

Comment:
```suggestion
        and "`pandas API documentation for " not in target_doc
```

Reply:
@anmyachev pls make this change next time you'll be fixing something in there (I think line numbers are still off, so they need to be fixed, right?)

Reply:
Yes, line numbers need to be fixed.